doc/cephfs/fs-nfs-exports.rst

```diff
@@ -0,0 +1,62 @@
+=======================
+CephFS Exports over NFS
+=======================
+
+CephFS namespaces can be exported over NFS protocol using the
+`NFS-Ganesha NFS server <https://github.com/nfs-ganesha/nfs-ganesha/wiki>`_.
+
+Requirements
+============
+
+-  Latest Ceph file system with mgr and dashboard enabled
+-  'nfs-ganesha', 'nfs-ganesha-ceph' and nfs-ganesha-rados-grace packages
+   (version 2.7.6-2 and above)
+
+Create NFS Ganesha Cluster
+==========================
+
+.. code:: bash
+
+    $ ceph nfs cluster create <type=fs> [--size=1] <clusterid>
```

Comment:
```suggestion
    $ ceph nfs cluster create <type=cephfs> [--size=1] <clusterid>
```

minor nit, but would `type=cephfs` make more sense in this context?

Reply:
Done.